### ISSUE TITLE
face recognition

### ISSUE THREAD

**Issue by myinzack** [flyyufelix/DenseNet-Keras#10]
By leveraging on this denseNet-Keras framework  can we undergo face recognition.
if so, what is the accuracy we can achieve ?

**Comment by flyyufelix** [comment]
No, models pretrained on ImageNet cannot perform face recognition directly. You will have to fine-tune the model with your own dataset of faces. If what you want is to detect face in the images, you can use OpenCV Haar Cascades (http://docs.opencv.org/trunk/d7/d8b/tutorial_py_face_detection.html). If you wish to perform face recognition/classification/clustering, you can use the pretrained models provided in OpenFace (http://cmusatyalab.github.io/openface/models-and-accuracies/).
